server/apps/immich/src/api-v1/asset/asset.service.ts

```diff
@@ -286,9 +287,9 @@ export class AssetService {
 
         await fs.access(videoPath, constants.R_OK | constants.W_OK);
 
-        if (query.isWeb && asset.mimeType == 'video/quicktime') {
+        if (query.isWeb) {
           videoPath = asset.encodedVideoPath == '' ? String(asset.originalPath) : String(asset.encodedVideoPath);
-          mimeType = asset.encodedVideoPath == '' ? asset.mimeType : 'video/mp4';
+          mimeType = asset.encodedVideoPath == '' ? asset.mimeType : `video/${extname(asset.encodedVideoPath!)}`;
```

Comment:
If we use the extension as MIME type, I don't think it will be correct for all cases. For example, `.mov` has MIME type of `video/quicktime`

Reply:
Yea we definitely can't do that

Reply:
Then, we'd need to store that info in the DB as well